openapi-v1.yaml

```diff
@@ -6322,59 +6541,240 @@ paths:
+          schema:
+            $ref: "#/definitions/Error"
+
+  /groups/browser/public:
+    get:
+      description: Returns one page of public groups.
+      operationId: listPublicGroups
+      tags:
+        - groups
+      parameters:
+        - name: page
+          in: query
+          description: pagination offset
+          type: integer
+          required: false
+        - name: per_page
+          in: query
+          description: pagination limit
+          type: integer
+          required: false
+        - name: search
+          in: query
+          description: search string that will look at name, namespace or description fields
+          required: false
+          type: string
+        - name: namespace
+          in: query
+          description: namespace
+          required: false
+          type: string
+        - name: orderby
+          in: query
+          description: sort by which field valid values include last_accessed, size, name
+          required: false
+          type: string
+        - name: permissions
+          in: query
+          description: permissions valid values include read, read_write, write, admin
+          required: false
+          type: string
+        - name: tag
+          in: query
+          description: tag to search for, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
+        - name: exclude_tag
+          in: query
+          description: tags to exclude matching array in results, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
+        - name: flat
+          in: query
+          description: if true, ignores the nesting of groups and searches all of them
+          type: boolean
+        - name: parent
+          in: query
+          description: search only the children of the groups with this uuid
+          type: string
+          required: false
+      responses:
+        200:
+          description: the group contents
+          schema:
+            $ref: '#/definitions/GroupBrowserData'
+        default:
+          description: error response
+          schema:
+            $ref: "#/definitions/Error"
+
+  /groups/{group_namespace}/{group_name}/contents:
     parameters:
-      - name: namespace
+      - name: group_namespace
         type: string
         in: path
         required: true
         description: The namespace of the group
-      - name: name
+      - name: group_name
         type: string
         in: path
         required: true
-        description: The name of the group
+        description: The unique name or id of the group
     post:
-      description: Creates a new, empty group in the namespace.
-      operationId: createGroup
+      description: Changes the contents of the group by adding/removing members.
```

Comment:
@alekshs should this be a patch to the object directly?

Reply:
I think this is ok since we use the root path to patch GroupInfo (name, description, tags etc). Contents do not belong to the GroupInfo model.